src/config/config_chicago.yml

```diff
@@ -0,0 +1,108 @@
+# City name
+city: Chicago,IL,USA
+# City centerpoint latitude & longitude (default geocoded values set)
+city_latitude: 41.884250000000065
+city_longitude: -87.63244999999995
+
+# City's time zone: defaults to the local time zone of computer initializing the city's config file
+timezone: America/Chicago
+# Radius of city's road network from centerpoint in km, required if OSM has no polygon data (defaults to 20km)
+city_radius: 20
+speed_unit: kph
+
+# By default, maps are created from OSM's polygon data and fall back to radius
+# if there is no polygon data, but but you can change the openstreetmap_geography
+# to 'radius' if preferred
+map_geography: shapefile
```

Comment:
Did updating the osmnx version fix the polygon issue? If so, did still you prefer your predefined polygon to osmnx's?

Reply:
yeah, I had the same polygon issue with the predefined shape file, that was only resolved with the osmnx upgrade. I haven't gone back to checking against the polygon pulled from nominatim, but the shapefile I used is the one provide by the city's data portal.